examples/layers/raw/canvas.dart

```diff
@@ -70,10 +70,10 @@ ui.Picture paint(ui.Rect paintBounds) {
 
   // Scale x and y by 0.5.
   final Float64List scaleMatrix = Float64List.fromList(<double>[
-      0.5, 0.0, 0.0, 0.0,
-      0.0, 0.5, 0.0, 0.0,
-      0.0, 0.0, 1.0, 0.0,
-      0.0, 0.0, 0.0, 1.0,
+    0.5, 0.0, 0.0, 0.0, //
```

Comment:
Did an empty comment get added here?

Reply:
Yes. I added it. It's a common trick to drive `dartfmt` to prevent putting every element on a new line.